src/components/SignInPage/SigninPage.scss

```diff
@@ -0,0 +1,47 @@
+.signin-page-wrapper {
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    width: 100%;
+    height: 100vh;
+
+    .signin-central-window {
+        width: 800px;
+        display:flex;
+        flex-direction: column;
+        align-content: center;
+        justify-content: center;
+        background-color: white;
+        padding: 25px;
+        border-radius: 25px;
+
+        header {
+            font-size: 1.75rem;
+        }
+
+        .signin-input-wrapper{
+            text-align: left;
+            margin: 10px 10px;
+
+            label {
+                padding-left: 10px;
+            }
+        }
+
+        footer {
+            button {
+                margin-top: 10px;
+            }
+
+            p {
+                margin-top: 25px;
+            }
+
+            p:hover {
```

Comment:
Why you create :hover if it always be pointer? You can use it on main class instead.